expo/gbe/templates/gbe/profile/incl_performer.tmpl

```diff
@@ -1,78 +1,26 @@
 <div class='performer_section'>
   {% if personae or troupes or combos %}
-    <span class='heading performers'> 
+    <span class='heading performers'>
       <b>Performer Items</b>
     </span>
-    <br>
+  <br>
     {% if personae or troupes or combos %}
       {% for performer in personae %}
-        <div class='performer'>
-	{% if performer.promo_thumb %}
-          <div class="gbe_thumb_block">
-	    <img src="{{MEDIA_URL}}{{performer.promo_thumb}}" class="gbe_thumbnail">
-	  </div>
-	{% else %}
-	  <div class="gbe_no_pic">&nbsp;</div>
-	{% endif %}
-	  <div class="performer_text">
-  	    <a href='{% url 'gbe:persona_edit' performer.resourceitem_id %}'>
-            {{ performer }}
-     	    </a><br>
-	    (Click to edit) 
-          </div>
-	</div>
-        {% for alert in performer.alerts %}
-          <span class='alert'>{{alert | safe}} </span><br>
-        {% endfor %}
+        {% with 'gbe:persona_edit' as edit_view_name %}
+        {% include 'gbe/profile/incl_performer_block.tmpl' %}
+	{% endwith %}
       {% endfor %}
       {% for performer in troupes %}
-        <div class='performer'>
-	{% if performer.promo_thumb %}
-          <div class="gbe_thumb_block">
-	    <img src="{{MEDIA_URL}}{{performer.promo_thumb}}" class="gbe_thumbnail">
-	  </div>
-	{% else %}
-	  <div class="gbe_no_pic">&nbsp;</div>
-	{% endif %}
-	  <div class="performer_text">
-          {% if performer.contact = profile %}
```

Comment:
Where is the analog of this line in the refactored code?

BLOCKER:  With the change in the code, I can now edit troupes in which I am a *member* but not the owner.  In master, I can *view* troupes where I am a member.  This is a regression.